doc/source/contribution_and_api/unit_testing.rst

```diff
@@ -0,0 +1,42 @@
+.. _ref_unit_testing_contributing:
+
+Unit Testing
+===============
+
+Unit testing is highly important. It enables to check that the
+changes on the code are efficient and don't have an impact on
+other parts of the code. 
+
+Unit tests are in the ``tests`` directory inside this repository.
+
+Test Validation GitHub workflow
+-------------------------------
+
+`Codecov <https://github.com/codecov>`_ generates a coverage report of
+the committed code. It details how merging the Pull Request will impact the
+coverage. It is one of the checks that must be approved to
+merge the code.
+
+Creating a unit test 
+--------------------
+
+In the pymapdl repository, the tests are done with the **pytest package**. 
+
+To create a pytest file, its name must be one of the following forms:
+``filename_test.py`` or ``test_filename.py``.
+
+To create tests, please take note of the `pytest documentation <https://docs.pytest.org/en/7.2.x/>`_ .
```

Comment:
I would like you to expand this a bit more... the pymapdl documentation should be as self-complete as possible, but without falling into repetition. I know keep an equilibrium on this is complicated, but ome info about how to do tests should be included. Leaving up to the user check for details on the pytest website.

Reply:
Hello German!
I have added some examples to make it clearer. Let me know if you want me to add further explanations.